js_modules/dagit/packages/ui/src/components/TokenizingField.tsx

```diff
@@ -70,7 +70,8 @@ export function tokenizedValueFromString(
   str: string,
   providers: SuggestionProvider[],
 ): TokenizingFieldValue {
-  const [token = '', value = ''] = str.split(':');
+  const [token = '', ...remaining] = str.split(':');
```

Comment:
micro-optimization: `const [_, token, value] = 'token:some:value'.match(/([^:]+)(.+$)/)` to skip the intermediate array if there is a `:` in the value.

Reply:
Interesting! I think the regex would make it slightly harder to understand (especially since the array returned by `match`  is such a weird object) but now that you mention it, I think I could do an `indexOf` with string slicing to skip the array.